Assets/Animations.meta

```diff
@@ -1,9 +0,0 @@
-fileFormatVersion: 2
```

Comment:
Unity refuses me to keeps these files 😞 According to the log messages, the meta files cannot exist with the associated asset.